micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/DigitJSONForm.js

```
@@ -255,110 +278,215 @@ const DigitJSONForm = ({
+    }
+  }
+  onSubmit && onSubmit(transformedFormData, additionalProperties);
+};
+
+
+  const customWidgets = { SelectWidget: v2 ? CustomDropdown : CustomDropdownV2, CheckboxWidget: CustomCheckbox };
 
   const onError = (errors) => {
-    setLiveValidate(true);
-    onFormError(errors);
+    onFormError && onFormError(errors);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)